src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -230,6 +230,7 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
             } else if (method === 'explore') {
                 exploreControl.setFromUserPreferences();
                 tabControl.setTab(tabControl.TABS.EXPLORE);
+                exploreControl.getNearbyPlaces();
```

Comment:
:+1: Yup, this is better than what I was suggesting.  I don't see any remaining problems when moving between views.